frontend/src/test/LoginPage.test.js

```diff
@@ -1,13 +1,10 @@
-import React from 'react';
 import { create } from 'react-test-renderer';
-import { BrowserRouter } from 'react-router-dom';
-
-
-import LoginPage from '../Components/LoginPage';
+import LoginPage from '../Components/LoginRegPage';
 
 describe('Login Page Snapshot tests',()=>{
     test(`Login Page matches snapshot`, () => {
         const loginPage = create(LoginPage);
```

Comment:
The create method takes in the component itself, so this should be create(<LoginPage/>)

Reply:
What do you mean?

Reply:
The create method should take `<LoginPage/>` instead of `LoginPage`.

Reply:
I think you meant to say that replace with <LoginRegPage/>. I have done that and tested that tests pass. Have another look to see if something needs changing

Reply:
Yh I meant replace with. Everything else seems good.